hail/python/hailtop/hailctl/config/cli.py

```diff
@@ -49,7 +49,7 @@ def complete_config_variable(incomplete: str):
             yield (var.value, var_info.help_msg)
 
 
-@app.command()
+@app.command(help='Set the value of a configuration parameter.')
```

Comment:
How come you needed to add these help texts? Is it not using the function's docstring?

Reply:
Ah. Didn't realize that was what was happening. I removed them.